apps/userdashboard/assets/js/a4_candy_userdashboard/__tests__/AiReportExplanation.jest.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { render, fireEvent, screen } from '@testing-library/react'
+import '@testing-library/jest-dom'
+import { AiReportExplanation } from '../AiReportExplanation'
+
+test('Snapshot Read More <AiReportExplanation>', () => {
+  const { container } = render(
+    <AiReportExplanation AiReport="This is the ai report" />
+  )
+  expect(container).toMatchSnapshot()
+})
+
+test('Snapshot Read Less <AiReportExplanation>', () => {
+  const { container } = render(
+    <AiReportExplanation AiReport="This is the ai report" />
+  )
+  const button = screen.getByRole('button')
+  fireEvent.click(button)
+  expect(container).toMatchSnapshot()
+})
```

Comment:
@goapunk how are these testings the read more or less? they don't seem to check if the text changes, unless I am missing something?